src/applications/vaos/referral-appointments/ScheduleReferral.unit.spec.js

```diff
@@ -1,35 +1,36 @@
 import React from 'react';
 import { expect } from 'chai';
-import sinon from 'sinon';
-import ScheduleReferral from './ScheduleReferral';
 import {
   createTestStore,
   renderWithStoreAndRouter,
 } from '../tests/mocks/setup';
-import { referral } from './temp-data/referral';
+import ScheduleReferral from './ScheduleReferral';
+import { createReferral } from './utils/referrals';
 
 describe('scheduleReferral component', () => {
   it('should display the subtitle correctly given different numbers of appointments', async () => {
```

Comment:
This should be `...given one appointment`

Reply:
I'll leave those changes to this other ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/95270